### PR TITLE
Rearranged folder structure of the contributors site

### PR DESCRIPTION
**Pull request by Atharva-Shinde**
#### Changes:
- moved `events/community-meeting.md` to `community/community-meeting.md`
- moved `events/meet-our-contributors.md` to `community/meet-our-contributors.md`
- moved `events/office-hours.md` to `community/office-hours.md`
- moved `events/past-events/2020` to `events/2020`
- moved `events/past-events/2021` to `events/2021`
- created folder `events/2022/_index.md`  and moved `events/kcseu` to `events/2022/kcseu`

#### screenshot of rearranged folders
Screenshot 2022-07-26 at 4 12 43 PM

#### screenshots of website
`/community`
Screenshot 2022-07-26 at 4 14 05 PM

`/events`
Screenshot 2022-07-26 at 4 15 41 PM

### CODE REVIEW

File: content/en/community/community-meeting.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+---
+title: Monthly Community Meeting
```
Comment (mrbobbytables) [inline content/en/community/community-meeting.md:2]: The community meeting, office-hours and, moc docs are external content and pulled in externally through the [hack/gen-content.sh](https://github.com/kubernetes/contributor-site/blob/master/hack/gen-content.sh) script.

To permanently remove them, they will need to be modified here:

https://github.com/kubernetes/contributor-site/blob/5500385f6d00cf7682b4aa8ece81c089595e418a/external-sources/kubernetes/community#L17-L19

We want to retain the community meeting, its new location should be under community. The other 2 (office-horus/moc) can be removed. They aren't something we're actively doing these days. 👍 

We will also need to remove the `.gitignore` file in the events dir, its no longer adding anything:
https://github.com/kubernetes/contributor-site/blob/master/content/en/events/.gitignore

and add community-meeting.md to the .gitignore in the community dir
https://github.com/kubernetes/contributor-site/blob/master/content/en/community/.gitignore


As an aside, a follow up PR should be done to the community meeting docs in k/community to change its weight to 3 to position it better under the community section.

File: content/en/events/2020/kcc2020/_index.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+   contributors!
+type: docs
+---
+
+{{% alert title="Notice" color="warning" %}}
+This event has passed. Recordings from the event can be found on the
+[Kubernetes Community YouTube Channel](https://youtube.com/playlist?list=PL69nYSiGNLP0MPeiM9oAffsxtrUgUAu1g).
+{{% /alert %}}
+
+<img align="center" alt="you're invited" style="max-width:50%;" src="/events/past-events/2020/kcc2020/invited.png">
```
Comment (mrbobbytables) [inline content/en/events/2020/kcc2020/_index.md:20]:
```suggestion
<img align="center" alt="you're invited" style="max-width:50%;" src="/events/2020/kcc/invited.png">
```

This is going to be the more annoying part of the migration, ALL the links have to be updated. -_- I can also take care of this and push to your branch once the files are in the new location 👍 

we can also drop the 2020 from this directory title as its now a native part of the url.

**Review by mrbobbytables — CHANGES_REQUESTED** [review]
Thanks for kicking this off :)
I made a few comments - It's going to be a little more complicated as some of the content is external and managed through the ingestion script. \~_\~

The links are also going to need to be updated. with the location change =/

**Comment by jberkus** [comment]
More changes:

We do NOT need _index.md pages under each year (i.e. `events/2022/_index.md`).

We DO need a new _index.md page under the main events folder (i.e. `events/_index.md`).   This page should have a tabular listing of current and past events.

The latter could be added in a later pull request, but if so this PR shouldn't fix the issue.

**Comment by mrbobbytables** [comment]
We do probably need an _index.md in each year to have the weight in the frontmatter, otherwise the oldest year will always be at the top instead of the most recent year =/

**Comment by jberkus** [comment]
Bob: oh, point, comment fixed.

File: content/en/community/community-meeting.md (hunk 1 above)
Comment (Atharva-Shinde) [reply content/en/community/community-meeting.md:2]: whats the role of .gitignore here? for eg does `.gitignore` excludes `community-meeting.md` from something while building the website?

File: content/en/events/2020/kcc2020/_index.md (hunk 2 above)
Comment (Atharva-Shinde) [reply content/en/events/2020/kcc2020/_index.md]: I just saw it, there are lots of links that needs to be updated :0

File: content/en/community/community-meeting.md (hunk 1 above)
Comment (mrbobbytables) [inline content/en/community/community-meeting.md:2]: This doesn't need to be committed. It'll be synced automatically when the site is generated 👍

File: content/en/community/meet-our-contributors.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
+---
+title: Meet our Contributors
```
Comment (mrbobbytables) [inline content/en/community/meet-our-contributors.md:2]: This file should be removed, we're no longer syncing it.

File: content/en/community/office-hours.md [hunk 4]
```diff
@@ -0,0 +1,184 @@
+---
+title: Office Hours
```
Comment (mrbobbytables) [inline content/en/community/office-hours.md:2]: This file should be removed, we're no longer syncing it.

File: content/en/events/2020/kcc2020/_index.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
+---
```
Comment (mrbobbytables) [inline content/en/events/2020/kcc2020/_index.md:1]: We can drop 2020 from the directory name, the file path should look like this:
`content/en/events/2020/kcc/_index.md`

File: content/en/events/2020/kcc2020/faq.md [hunk 6]
```diff
@@ -0,0 +1,211 @@
...
+
+Discord server invites will be sent out closer to the event.
+
+
+### What sort of activities are planned?
+
+There will be trivia and other games available. For more information see the
+[activities] page.
+
+[activities]: /events/kcc2020/activities/
```
Comment (mrbobbytables) [inline content/en/events/2020/kcc2020/faq.md:81]: every link needs to be updated to reflect the new file paths.
Find -> replace
`/events/kcc2020` -> `/events/2020/kcc`

File: content/en/events/2021/kcc2021/_index.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
+---
+title: "Contributor Celebration"
```
Comment (mrbobbytables) [inline content/en/events/2021/kcc2021/_index.md:2]: We can drop 2021 from the directory path. Path should look like this
`content/en/events/2021/kcc2021/_index.md` -> `content/en/events/2021/kcc/_index.md`

File: content/en/events/2021/kcc2021/faq.md [hunk 8]
```diff
@@ -0,0 +1,223 @@
...
+
+Discord server invites will be sent out closer to the event.
+
+
+### What sort of activities are planned?
+
+There will be trivia and other games available. For more information see the
+[activities] page.
+
+[activities]: /events/past-events/2021/kcc2021/activities/
```
Comment (mrbobbytables) [inline content/en/events/2021/kcc2021/faq.md:80]: All links need to be updated
Find -> replace
`/events/past-events/2021/kcc2021` -> `/events/2021/kcc`

File: content/en/events/2021/kcsna/_index.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+title: "Contributor Summit North America 2021"
+type: docs
+aliases: [ "/kcsna2021" ]
+weight: 1
+description: >
+  The Kubernetes Contributor Summits are returning for 2021!
+
+---
+
+<img align="center" alt="Kubernetes Contributor Summit in Los Angeles, California" style="max-width:80%;" src="/events/past-events/2021/kcsna/banner.png">
```
Comment (mrbobbytables) [inline content/en/events/2021/kcsna/_index.md:11]: All file paths needs to be updated
Find -> Replace
`/events/past-events/2021/kcsna` -> `/events/2021/kcsna`

File: content/en/events/2022/kcseu/_index.md [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+
+### About
+
+The Europe Summit will be a lightweight event taking place on May 16th
+in Valencia, Spain alongside 
+<a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/" rel="noopener noreferrer" target="_blank">KubeCon</a>
+at the
+<a href="https://www.feriavalencia.com/en/" rel="noopener noreferrer" target="_blank">Feria Valencia</a>.
+
+The Summit will be a loosely organized event with a [relaxed schedule](/events/kcseu/schedule),
```
Comment (mrbobbytables) [inline content/en/events/2022/kcseu/_index.md:20]: All file paths need to be updated
Find -> Replace
`/events/kcseu` -> `/events/2022/kcseu`

**Review by mrbobbytables — CHANGES_REQUESTED** [review]

**Review by mrbobbytables — COMMENTED** [review]
Thanks! Getting really close, noticed 2 things I didn't catch in previous reviews and  one new thing - the `hugo_build.lock` file committed, can you remove it and add an entry to the root .gitignore? Those definitely shouldn't be committed and I'm surprised we haven't encountered one accidentally being included before :x

File: content/en/events/contributor-summits.md [hunk 11]
```diff
@@ -3,7 +3,7 @@ title: Contributor Summits
 description: >
   Contributor summits bring together new and current Kubernetes contributors to
   connect and learn from one another.
-weight: 2
+weight: 4
```
[the remark this answers is not in the document]
Comment (Atharva-Shinde) [reply content/en/events/contributor-summits.md]: done!

File: external-sources/kubernetes/community [hunk 12]
```diff
@@ -14,9 +14,7 @@
 "/contributors/guide/release-notes.md","/docs/guide/release-notes.md"
 "/contributors/guide/git_workflow.png","/docs/guide/git_workflow.png"
 "/contributors/guide/contributor-cheatsheet/README.md","/docs/contributor-cheatsheet.md"
-"/events/community-meeting.md","/events/community-meeting.md"
-"/events/office-hours.md","/events/office-hours.md"
-"/mentoring/programs/meet-our-contributors.md","/events/meet-our-contributors.md"
+"/community/community-meeting.md","/community/community-meeting.md"
```
Comment (mrbobbytables) [inline external-sources/kubernetes/community:17]:
```suggestion
"/events/community/community-meeting.md","/community/community-meeting.md"
```
One last thing - the original location shouldn't change, just the destination 👍

File: external-sources/kubernetes/community [hunk 13]
```diff
@@ -14,9 +14,7 @@
 "/contributors/guide/release-notes.md","/docs/guide/release-notes.md"
 "/contributors/guide/git_workflow.png","/docs/guide/git_workflow.png"
 "/contributors/guide/contributor-cheatsheet/README.md","/docs/contributor-cheatsheet.md"
-"/events/community-meeting.md","/events/community-meeting.md"
-"/events/office-hours.md","/events/office-hours.md"
-"/mentoring/programs/meet-our-contributors.md","/events/meet-our-contributors.md"
+"/events/community/community-meeting.md","/community/community-meeting.md"
```
Comment (mrbobbytables) [inline external-sources/kubernetes/community:17]:
```suggestion
"/events/community-meeting.md","/community/community-meeting.md"
```
messed up my comment, fixing.

**Review by mrbobbytables — APPROVED** [review]
okay, I think that'll do it :)
/approve
/hold
 for sign off from @jberkus 
/label tide/merge-method-squash

**Review by jberkus — APPROVED** [review]

**Pull request merged**